angular/src/app/containers/leaflet/leaflet.component.ts

```diff
@@ -29,11 +47,76 @@ export class LeafletComponent implements AfterViewInit {
         bounds: config.maxBounds
       });
 
-      baseLayer.addTo(leafletMap);
+      baseLayer.addTo(this.leafletMap);
 
       setTimeout(() => {
-        leafletMap.invalidateSize();
+        this.leafletMap.invalidateSize();
       });
+
+      this.leafletMap.on('moveend', this.updateBoundingBox.bind(this));
```

Comment:
Ja, dat is een goeie. Ik denk dat het wel kan als je zorgt dat je `this.leafletMap.getBounds().toBBoxString()` meegeeft aan de reducer. Die moet dan zorgen voor de API calls en de data moet dan uiteindelijk weer op de state komen.
Het gaat me nu alleen te ver om dat voor deze PR te doen.